### PR TITLE
data extraction kit update

### PR DESCRIPTION
**Pull request by snova-luiss**
Update langchain versions requirement files
Update HuggingFaceEmbeddings from langchain community docker files
Fix syntax error and add select llm expert - retrieval_from_pdf_tables notebook

### CODE REVIEW

File: data_extraction/notebooks/retrieval_from_pdf_tables.ipynb [hunk 1]
```diff
@@ -552,7 +552,7 @@
     "load_dotenv(os.path.join(repo_dir,'.env'))\n",
     "\n",
     "llm = SambaNovaEndpoint(\n",
-    "    model_kwargs={\"do_sample\": False, \"temperature\": 0.0},\n",
+    "    model_kwargs={\"do_sample\": False, \"temperature\": 0.0, \"select_expert\": \"llama-2-7b-chat-hf\"},\n",
```
Comment (snova-jorgep) [inline data_extraction/notebooks/retrieval_from_pdf_tables.ipynb:555]: here please add a comment that indicates select expert should be removed if not using CoE

**Review by snova-jorgep — CHANGES_REQUESTED** [review]
minor change requested

**Review by snova-jorgep — APPROVED** [review]

**Comment by snova-jorgep** [comment]
@snova-kwasia already reviewed by me, can we merge this PR

**Comment by snova-kwasia** [comment]
Thanks @snova-luiss great work. Merged.

**Pull request merged**